src/js/components/CheckBox/StyledCheckBox.js

```diff
@@ -45,7 +45,11 @@ Object.setPrototypeOf(StyledCheckBoxIcon.defaultProps, defaultProps);
 const StyledCheckBoxContainer = styled.label`
   display: flex;
   flex-direction: row;
-  align-items: center;
+  align-items: ${props =>
+    (props.theme.checkBox &&
+      props.theme.checkBox.label &&
+      props.theme.checkBox.label.align) ||
+    'center'};
```

Comment:
Since in the docs it says it defaults to center, do you think we should move this to base.js as the default value. Otherwise, if someone looks at base.js and sees undefined it might be confusing.